.yamato/yamato-ci.yml

```diff
@@ -0,0 +1,67 @@
+linux:
+  name: "Linux"
+  agent:
+    type: Unity::VM
+    image: cds-ops/cds-ubuntu-18.04-base:stable
+    flavor: b1.small
+  commands:
+    - git submodule init
+    - git submodule update
+    - mono bee.exe build::tundra2::master
+    - mono bee.exe run::tundra2-unittest::master
+  artifacts:
+    artifacts:
+      paths:
+        - "build/**/*"
+mac:
+  name: "Mac"
+  agent:
+    type: Unity::VM::osx
+    image: buildfarm/mac:stable
+    flavor: m1.mac
+  commands:
+    - git submodule init
+    - git submodule update
+    - mono bee.exe build::tundra2::master
+    - mono bee.exe run::tundra2-unittest::master
+  artifacts:
+    artifacts:
+      paths:
+        - "build/**/*"
+windows:
+  name: "Windows"
+  agent:
+    type: Unity::VM
+    image: cds-ops/win10-base:stable
+    flavor: b1.small
+  commands:
+    - git submodule init
+    - git submodule update
+    - bee.exe build::tundra2::master
+    - bee.exe run::tundra2-unittest::master
+  artifacts:
+    artifacts:
+      paths:
+        - "build/**/*"
+builds-zip:
+  name: "zPackage"
```

Comment:
Could we have better names for these builders (like 'Build + Test Windows')? And it's 4 jobs, do we really need 'z'-prefixes?